migrations.lock

```diff
@@ -337,6 +337,8 @@ data_interfaces
  0032_bootstrap_audit_events_for_update_rules
  0033_automaticupdaterule_deleted_on
  0034_case_name_actions
+ 0035_dedupe_add_match_values
+ 0036_populate_dedupe_match_values
```

Comment:
Can these be combined into a single migration? Migrations are relatively expensive, so the less we have the better.